build/BuildNuGets.csx

```diff
@@ -27,17 +28,30 @@ foreach (var file in files)
 {
     var nugetArgs = $@"pack {file} " +
         $@"-BasePath ""{binDir}"" " +
+        $@"-OutputDirectory ""{outDir}"" " +
         "-ExcludeEmptyDirectories " +
         $@"-prop licenseUrl=""{licenseUrl}"" " +
         $@"-prop version=""{buildVersion}"" " +
         $"-prop authors={authors} " +
         $@"-prop projectURL=""{projectURL}"" " +
         $@"-prop tags=""{tags}""";
-    Console.WriteLine(nugetArgs);
-    var p = Process.Start(Path.GetFullPath("../../nuget.exe"), nugetArgs);
+    var nugetExePath = Path.GetFullPath("../../nuget.exe");
+    var p = new Process();
+    p.StartInfo.FileName = nugetExePath;
+    p.StartInfo.Arguments = nugetArgs;
+    p.StartInfo.UseShellExecute = false;
+
+    Console.WriteLine($"Running: nuget.exe {nugetArgs}");
+    p.Start();
+    procs.Add(p);
 }
 
+int exit = 0;
+
 foreach (var p in procs)
 {
     p.WaitForExit();
```

Comment:
We don't redirect output so it should be printed to the screen (I have manually tested this).